File: Backend/MarketPlace_Backend/marketplace/models/productor.py
from django.db import models
from .user import User


class Productor(models.Model):
    direccion = models.CharField(max_length=200, null=False, blank=False)
    fotoProveedor = models.CharField(max_length=500, blank=False, null=False)
    numeroTelefono = models.CharField(max_length=20, null=False, blank=False)
    descripcion = models.CharField(max_length=200, null=False, blank=False)
    user = models.OneToOneField(User, on_delete=models.CASCADE)

File: src/app/usuario/productor/productor.ts
import {Usuario} from "../cliente/usuario";
import {Oferta} from "../../catalogo/oferta/oferta";

export class Productor extends Usuario {
  id: number;
  direccion: string;
  fotoProveedor: string;
  activado: boolean;
  numeroTelefono: string;
  descripcion: string;
  ofertasProductor: Oferta[];
}
